balancer/grpclb/grpclb_util.go

```diff
@@ -62,12 +62,25 @@ import (
 type lbManualResolver struct {
```

Comment:
Oh... this has its own manual resolver too?  Can we reuse the one in `resolver/manual`?

If not, we probably should fix that to be tolerant to idleness by setting `bootstrapState` here: https://github.com/grpc/grpc-go/blob/62726d4336c675cb45b65cded43838a85f1cbc25/resolver/manual/manual.go#L102

Reply:
Changed it to use the `manual` resolver, but we still need this `lbManualResolver` type since it provides some level of wrapping of the one particular piece of functionality that is provided here. Otherwise, we will have to move all this functionality to the top-level `lbBalancer` type and also store the `resolver.ClientConn` in there.

Reply:
> it provides some level of wrapping of the one particular piece of functionality that is provided here

...which is what, specifically?  Calling `ResolveNow` on the `balancer.ClientConn` when the `ResolveNowCallback` is invoked?  Does that necessitate a wrapping type though?

Reply:
You are right. We don't need this type at all. Got it to work by just using the `manual.Resolver`.